tests/www/invitations_views/test_siae_accept.py

```diff
@@ -309,7 +309,7 @@ def test_accept_existing_user_already_member_of_inactive_siae(self):
         self.client.force_login(user)
         response = self.client.get(invitation.acceptance_link, follow=True)
         self.assertRedirects(response, reverse("welcoming_tour:index"))
-        # /invitations/<uui>/join_siae then /welcoming_tour/index
+        # /invitations/<uui>/join_company then /welcoming_tour/index
```

Comment:
```suggestion
        # /invitations/<uui>/join-company then /welcoming_tour/index
```
(si on change l'URL)